config/config.yaml

```diff
@@ -0,0 +1,68 @@
+github:
+  token: "secret:spinnaker-community/spinbot-gh-token#latest"
+
+logging:
+  level: INFO
+
+storage:
+  gcs:
+    bucket: spinbot-cache
+    project: spinnaker-community
+    path: cache
+
+policy:
```

Comment:
Do we need this anymore?  If we're using `stalebot` now I think we can delete this part of the config and the relevant code.

Reply:
(Though maybe we need to enable `stalebot` on the other repos here first.)

Reply:
This is primarily porting over current functionality. As you noted, there are other repos with issues that the current GHA doesn't look at.